include/cpp2util.h

```diff
@@ -746,6 +746,14 @@ auto to_string(std::optional<T> const& o) -> std::string {
     return "(empty)";
 }
 
+template<typename T>
+auto to_string(T const& t) -> std::string
+    requires (!std::is_array_v<T> // avoid ambiguity with to_string(std::string const&) in case of const char arrays
+        && std::is_same_v<T, std::string_view>)
+{
+    return {t.data(), t.size()};
+}
+
```

Comment:
Make it more like [this constructor](https://eel.is/c++draft/string.cons#lib:basic_string,constructor____).
```suggestion
template<typename T>
auto to_string(T const& t) -> std::string
    requires (std::is_convertible_v<const T&, std::string_view> && !std::is_convertible_v<const T&, const char*>)
{
    return std::string{t};
}

```

Reply:
It's the middle of the night for me... but Herb is reviewing the PRs so... I will fix it :)

This PR is part of my another one -> https://github.com/hsutter/cppfront/pull/40 (the complete support for standard types). It has a similar issue -> I will fix in both PRs)

Reply:
OK, I will correct it to this:

```cpp
template<typename T>
auto to_string(T const& sv) -> std::string
    requires (std::is_convertible_v<T, std::string_view> && !std::is_convertible_v<T, const char*>)
{
    return std::string{sv};
}
```

Reply:
Now it is fixed.

I have added examples to the regression tests as well.